config.py

```diff
@@ -139,7 +139,7 @@ class Development(Config):
     DM_API_AUTH_TOKEN = "myToken"
 
     DM_S3_ENDPOINT_URL = (  # use envvars to set this, defaults to using AWS
-        f"http://localhost:{os.environ['DM_S3_ENDPOINT_PORT']}"
+        f"http://s3.localhost.localstack.cloud:{os.environ['DM_S3_ENDPOINT_PORT']}"
         if os.getenv("DM_S3_ENDPOINT_PORT") else None
```

Comment:
I think this is only used to set DM_ASSETS_URL, so I'm not sure why the two are different in this commit?

Reply:
The S3 client in dmutils peaks at it: https://github.com/alphagov/digitalmarketplace-utils/pull/580/files

Though you're right that this is surprising behaviour. I'll take a look and see if I can move the logic into supplier-frontend

Reply:
Ah good point 🤦

Reply:
Updated. Once we've added this logic here and to admin frontend, we can remove it from dmutils.